src/components/datepicker/js/calendar.js

```diff
@@ -57,7 +57,8 @@
       scope: {
         minDate: '=mdMinDate',
         maxDate: '=mdMaxDate',
-        dateFilter: '=mdDateFilter'
+        dateFilter: '=mdDateFilter',
+        _currentView: '@mdCurrentView'
```

Comment:
I tried it out, but for some reason the datepicker kept resetting it after the calendar was opened. E.g. here's what I tried:

```js
var self = this;

console.log(self.currentView); // logs properly

setTimeout(function() {
  console.log(self.currentView); // logs an empty string 
});
```

It's weird, because it doesn't happen if I switch it to a two-way binding or I also add `this.currentView = this.currentView || 'month';` to the datepicker's controller. I went with the underscored property in order to keep all the `currentView` logic within the calendar.